src/pages/home/HowItWorksSection.tsx

```diff
@@ -51,37 +50,49 @@ export function HowItWorksSection() {
             <div className="grid gap-y-5">
               <FourthCol header={t`1. Get funded`}>
                 <Trans>
-                  Crowdfund your project with ETH. Set a funding target to cover
-                  predictable expenses, and any extra funds (
-                  <OverflowVideoLink>overflow</OverflowVideoLink>) can be
-                  claimed by anyone holding your project's tokens alongside you.
+                  Crowdfund your project with ETH. When someone pays your
+                  project, they'll receive your project's tokens (or NFTs) — use
+                  tokens to grant governance rights, community access, or other
+                  membership perks.
                 </Trans>
               </FourthCol>
-              <FourthCol header={t`2. Give ownership`}>
+              <FourthCol header={t`2. Build trust`}>
                 <Trans>
-                  When someone pays your project, they'll receive your project's
-                  tokens in return. Tokens can be redeemed for a portion of your
-                  project's overflow funds; when you win, your community wins
-                  with you. Leverage your project's token to grant governance
-                  rights, community access, or other membership perks.
+                  Transparently set your terms ahead of time, or take control
+                  when you need to. Juicebox lets you define elegant tokenomics,
+                  payouts, and other rules in advance, acting as a safeguard
+                  against rug pulls. Your supporters don't have to trust you —
+                  even though they already do.
                 </Trans>
               </FourthCol>
-              <FourthCol header={t`3. Manage your funds`}>
+              <FourthCol header={t`3. Stay flexible`}>
                 <Trans>
-                  Commit portions of your funds to the people or projects you
-                  want to support, or the contributors you want to pay. When you
-                  get paid, so do they.
-                </Trans>
-              </FourthCol>
-              <FourthCol header={t`4. Build trust`}>
-                <Trans>
-                  Changes to your project's funding configuration require a
-                  community-approved period to take effect, which acts as a
-                  safeguard against rug pulls. Your supporters don't have to
-                  trust you — even though they already do.
+                  As your community grows, scale your project with it: Juicebox
+                  lets you update your payouts, tokenomics, and other rules over
```

Comment:
**suggestion** i think we generally try to avoid `tokenomics` - wdyt about 'token configuration' or 'token behavior' or similar?

Reply:
Changed from tokenomics to "token issuance & redemption".